domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -78,6 +79,8 @@ def on_get_ep400_contentions(self, pending_contentions=None):
 
     @running_set_temp_station_of_jurisdiction.enter
     def on_set_temp_station_of_jurisdiction(self, pending_contentions=None, ep400_contentions=None):
```

Comment:
So on line 89 `pending_contentions` and `ep400_contentions` are both added to the self.process(...) function as kwargs. So they are actually available for you to use in the function signature of is_duplicate as kwargs: `is_duplicate(self, pending_contentions, ep400_contentions)`.  So you dont actually need to add them to the MergeJob class. 🪄 🧙

Reply:
I was thinking those might be useful to have for restarts/recovery when we get to converting the job store to RDS or some persistent backend. But we could definitely leave those out until we get to that point - thoughts?

Reply:
Noted! I'd say for now, lets just leave them out! We can have another conversation when it comes to the data we should/can/can't store in RDS!